alicloud/table_alicloud_rds_instance.go

```diff
@@ -768,7 +789,57 @@ func getRdsInstanceAkas(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
 	}
 	commonColumnData := commonData.(*alicloudCommonColumnData)
 	accountID := commonColumnData.AccountID
-	return []string{"acs:rds:" + region + ":" + accountID + ":instance/" + instanceID}, nil
+	return "acs:rds:" + region + ":" + accountID + ":instance/" + instanceID, nil
```

Comment:
check the arn format once - it should be -arn:acs:dds:cn-hangzhou:123456789xxxxxxx:instance/dds-uf6ab1exxxx